### PR TITLE
UF-305 - PatternFly updates

### PR DESCRIPTION
**Pull request by cristianonicolai**
Upgrade RCUE to version 3.6.0
Small style fixes post PatternFly 3.6.0 upgrade
Removed custom css for product color style
Add theme modules for community and product

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on:
https://github.com/uberfire/uberfire/pull/442
https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/240

File: kie-wb-theme/kie-wb-theme-community/pom.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <name>KIE Theme - Community</name>
+    <description>KIE Theme - Community</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.uberfire</groupId>
+            <artifactId>uberfire-workbench-client-views-patternfly</artifactId>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
```
Comment (psiroky) [inline kie-wb-theme/kie-wb-theme-community/pom.xml]: Nit picking...but XML files should use 2 spaces as indentation level.

**Comment by manstis** [comment]
+1 to merge (on face value.. IDK whether this PR is to be changed following discussion about the proliferation of GWT modules on https://github.com/uberfire/uberfire/pull/442).

One minor comment; the ```DOCTYPE``` for the new GWT modules are obviously copy and pasted.. we should be able to reference [GWT2.7](https://gwt.googlesource.com/gwt/+/2.7.0/distro-source/core/src/gwt-module.dtd)... but I guess it just highlights we've not updated any modules since 2.5 ;-)

**Comment by cristianonicolai** [comment]
@psiroky thanks for noticing that, turns out I had Intellij code style incorrectly set for XML's

@manstis DOCTYPE changed, regarding the changes on Uberfire, I've already updated the PR, so it should be good to go.

**Comment by manstis** [comment]
+1 to merge (previous ACK buried in comments).

@cristianonicolai I assume you'll coordinate actual merge of these PRs once all ACKs are received?

**Pull request merged**